src/irmin-pack/inode.ml

```diff
@@ -51,10 +50,14 @@ end
 
 module Make
     (Conf : CONFIG)
-    (Pack : Pack.MAKER)
-    (H : Irmin.Hash.S with type t = Pack.key)
+    (H : Irmin.Hash.S)
+    (Pack : Pack.MAKER
+            with type key = H.t
+             and type index = Pack_index.Make(H).t)
```

Comment:
do you need to expose this type equality here?

Reply:
Nope :) thanks.